src/ui/TimeSpanPicker.tsx

```diff
@@ -92,14 +90,15 @@ const TimeSpanPicker = ({
   onChangeStartTime,
   onChangeEndTime,
   disabled,
+  minWidth,
   ...props
 }: Props) => {
   const { t } = useTranslation();
   const idPrefix = `${id}-time-span-picker`;
 
   return (
     <Inline as="div" spacing={5} {...getInlineProps(props)}>
```

Comment:
```suggestion
    <Inline as="div" spacing={5} {...getInlineProps(props)}>
```

Can't you put the `minWidth` only on this Inline. Since you put it on both `Stack` children

Reply:
No since this will result of a min width of the parent container and not of a minWidth of the individual timespan elements.

Reply:
👍🏻